### ISSUE TITLE
Factory Block Placer does not place blocks.

### ISSUE THREAD

**Issue by DarkStarBuster** [stfwi/engineers-decor#176]
This issue appears to be similar to #160 in that the Factory Block Placer, in both the Overworld and Nether, fails to place a dirt block and instead drops the dirt as an item.
![image](https://user-images.githubusercontent.com/77248928/116091825-21148000-a69d-11eb-8f7f-0f0c45a53529.png)
![image](https://user-images.githubusercontent.com/77248928/116091907-38ec0400-a69d-11eb-889f-07c8e88af33b.png)

**Versions**
Forge: forge-1.16.5-31.1.0
Engineer's Decor: engineersdecor-1.16.4-1.1.11

Other mods are present as part of a modpack, details can be given further if needed.

**Comment by stfwi** [comment]
Hi, I quickly hacked an initial guess into the replaceability check of the placer, if you could double check with this jar. If it isn't that we need to dig deeper into the matter. Cheers!

[2021-04-27-engineersdecor-1.16.4-1.1.13-b3-test-placer-replace-check.zip](https://github.com/stfwi/engineers-decor/files/6387335/2021-04-27-engineersdecor-1.16.4-1.1.13-b3-test-placer-replace-check.zip)

**Comment by DarkStarBuster** [comment]
I replaced the 1.1.11 version with the jar you provided and booted up into  a creative world to test this.

I am happy to report that when placed and provided with a block, the block is properly placed. I am not able to test this on the world where I first noticed this issue due to it being a server outside of my control, however I am willing to report back again once the server has been updated with any new version with the fix contained in the jar you provided.
